components/playlist/playlist_download_request_manager.cc

```diff
@@ -71,6 +72,7 @@ void PlaylistDownloadRequestManager::CreateWebContents() {
     // |web_contents_| is created on demand.
     content::WebContents::CreateParams create_params(context_, nullptr);
     web_contents_ = content::WebContents::Create(create_params);
+    MarkAsBackgroundWebContents(web_contents_.get());
```

Comment:
If `PlaylistDownloadRequestManager` knows about all the background web_contents could we simply ask it in webPreferences calculation code? Instead of using `PlaylistBackgroundWebContentsTag`.